resolwe/flow/managers/dispatcher.py

```diff
@@ -416,6 +436,8 @@ def _prepare_executor(self, data_id, executor):
             be deployed.
         :rtype: (str, str)
         """
+        logger.debug(__("Preparing executor for data id {}", data_id))
```

Comment:
Similar as above, I would use `logger.debug(__("Preparing executor for Data with id {}", data_id))` here.